lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -355,6 +348,14 @@ class CustomEntityProperties(BaseModel):
         description="Routes to further information about the data",
         default_factory=FurtherInformation,
     )
+    audience: str = Field(
```

Comment:
Can we make this an enum? If it's free text then we'll end up with all sorts and it will be difficult to add conditional logic to the frontend